addons/popochiu/engine/objects/character/popochiu_character.gd

```diff
@@ -297,7 +297,7 @@ func queue_face_up_right() -> Callable:
 ## Makes the character to look up and right by setting [member _looking_dir] to [constant UP_RIGHT]
 ## and waits until [method idle] finishes.
 func face_up_right() -> void:
-	_looking_dir = Looking.UP_RIGHT
+	face_direction(position + Vector2(10, -10))
```

Comment:
This is clever but it can be more self-explanatory.

Vector2 has some constants, like `Vector2.RIGHT` and `Vector2.UP`. What about using

```suggestion
	face_direction(position + Vector2.UP + Vector2.RIGHT
```

or

```suggestion
	face_direction(position + (Vector2.UP + Vector2.RIGHT) * 10
```

It's just a small addition, but it makes it very readable.

Reply:
I like that solution much better!

Reply:
Updated - great idea!